tools/vol2obj/main.c

```diff
@@ -483,14 +482,18 @@ static bool _make_dir( const char* dir_path ) {
   return false;
 }
 
+/// Default string names for Volu video texture files.
+#define VOL_VID_STR_2048 "texture_2048_h264.mp4"
+#define VOL_VID_STR_1024 "texture_1024_h264.mp4"
```

Comment:
are we going to allow other formats in the future? can the tool accept arbitrary names?

Reply:
These filenames are just for dragging/importing a whole folder - it looks for these exact filenames first, since Volu exports these.
If you run via the command line you can specify any filename, and a variety of different file types, including webm, which we will almost certainly prefer in the future. I'd rather wait until we actually spit those out before changing the code though. I'm a little hesitant to encourage whole-folder scans since there are a number of avenues for mistakes and security issues, so I'm trying to keep this fairly limited.

Reply:
got it, thanks